docs/UserGuide.md

```diff
@@ -716,15 +716,15 @@ The code below is a direct quote from the default data file.
 The parameters in the braces indicate which parameter in the [Parameter descriptions](#parameter-descriptions) the input should adhere to.
 
 
-- `name`: Represents the student's name. If empty, input`""`. (NAME)
-- `phone`: Represents the student's phone number. If empty, input`""`. (PHONE)
-- `email`: Represents the student's email. If empty, input`""`. (EMAIL)
-- `address`: Represents the student's address. If empty, input`""`. (ADDRESS)
-- `remark`: Represents the remark or notes for a student and can contain the newline character `\n`. If empty, input`""`. (REMARK)
-- `education`: Represents the education level of a student. If empty, input`""`. (EDUCATION)
-- `telegram`: Represents the telegram handle of a student. If empty, input`""`. (TELEGRAM)
-- `modules`: Represents an array of modules the student takes. If empty, input`[ ]`. (MODULE)
-- `tagged`: Represents an array of tags the student is labelled with. If empty, input`[ ]`. (TAG)
+- `name`: Represents the student's name. Cannot be empty. (NAME)
+- `phone`: Represents the student's phone number. If empty, delete the entire "phone" row. (PHONE)
+- `email`: Represents the student's email. If empty, delete the entire "email" row. (EMAIL)
+- `address`: Represents the student's address. If empty, delete the entire "address" row. (ADDRESS)
+- `remark`: Represents the remark or notes for a student and can contain the newline character `\n`. If empty, input`""` or delete the entire "remark" row. (REMARK)
+- `education`: Represents the education level of a student. If empty, delete the entire "education" row. (EDUCATION)
+- `telegram`: Represents the telegram handle of a student. If empty, delete the entire "telegram" row. (TELEGRAM)
+- `modules`: Represents an array of modules the student takes. If empty, input `[ ]`, or `[]`, or delete the entire "module" row. (MODULE)
+- `tagged`: Represents an array of tags the student is labelled with. If empty, input `[ ]`, or `[]`, or delete the entire "tagged" row. (TAG)
```

Comment:
just say `[]` no need to mention that spaces can be inside the brackets, cuz thats more of a JSON thing.
instead of input `[]` say "set the value to"

also no we cannot delete the `tagged: []` or module row. it needs to be empty list